### PR TITLE
Add options to readme

### PR DESCRIPTION
**Pull request by gregurco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,15 +101,30 @@ eight_points_guzzle:
...
                     client_id:      "test-client-id"
                     client_secret:  "test-client-secret" # optional
                     scope:          "administration"
                     grant_type:     "Sainsburys\\Guzzle\\Oauth2\\GrantType\\ClientCredentials"
                     auth_location:  "body"
 ```
 
+### Options
+
+| Key | Description | Example |
```
Comment (florianpreusner) [inline README.md]: What do you think about a separate row "Required"?

Comment (gregurco) [reply README.md]: yep, I thought about that, but there are dependencies. For example username and password are required if grant type is "PasswordCredentials" and so on. If I will add column required then there will be a long description and not simply "yes/no"... :disappointed: Do you have any ideas?

Comment (florianpreusner) [reply README.md]: Tricky :/ Well, I don't have a perfect solution. One idea is to describe this by text, the other is to use different tables for each grant type.

**Review by florianpreusner — APPROVED** [review]
lgtm 👍

**Pull request merged**